01-syntax/04-high-order-functions/life-expectancy/lifeExpectancy.js

```diff
@@ -8,7 +8,21 @@ function average(array) {
 console.log(average([2, 2, 2]));
 // → 2
 
-// → Your code here.
+var ancestryByCentury = {};
```

Comment:
Use this one 
https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/life-expectancy/lifeExpectancy.js#L1

Hide all your work inside 'lifeExpectancy' function